### ISSUE TITLE
[BUG] Incorrect path in sap_media_download role

### ISSUE THREAD

**Issue by robfallows** [Azure/sap-hana#408]
**Describe the bug**

A generated path in `roles/sap_media_download/tasks/main.yml` breaks the playbook execution.

**To reproduce**

Execute the `sap_playbook.yml`:

```text
TASK [sap-media-download : Download installation media] ************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "python3 /home/<username>/sap-hana/deploy/v2/ansible../python/downloader/basket.py --config /home/<username>/output.json --dir /sapmnt", "delta": "0:00:00.025654", "end": "2020-04-14 11:57:56.301937", "msg": "non-zero return code", "rc": 2, "start": "2020-04-14 11:57:56.276283", "stderr": "python3: can't open file '/home/<username>/sap-hana/deploy/v2/ansible../python/downloader/basket.py': [Errno 2] No such file or directory", "stderr_lines": ["python3: can't open file '/home/<username>/sap-hana/deploy/v2/ansible../python/downloader/basket.py': [Errno 2] No such file or directory"], "stdout": "", "stdout_lines": []}
```

**Expected behavior**

Task succeeds without errors.

**Additional context**

Note the invalid path `/home/<username>/sap-hana/deploy/v2/ansible../python/downloader/basket.py`

This should be: `/home/<username>/sap-hana/deploy/v2/ansible/../python/downloader/basket.py`, although ideally it could be generated as `/home/<username>/sap-hana/deploy/v2/python/downloader/basket.py`

**Comment by ben-moss** [comment]
Seems to have been caused by this refactoring: https://github.com/Azure/sap-hana/pull/393/files#diff-5237f6e0310d3ead80ea2b990b34e74cR23

**Comment by rukawata** [comment]
Fixed with #414
